src/assets/md/oot100faq.md

```diff
@@ -0,0 +1,29 @@
+# Ocarina of Time Decomp 100% FAQ
+
+## What is this project's goal?
+
+We aim to produce a complete documented codebase, written in C as far as possible, for every version of The Legend of Zelda Ocarina of Time. This may be used for knowledge of the game (how its systems work, for glitches, for speedruns, etc.), or for changing parts of the game to make it act differently or do different things (that is, modding). This code compiles to be byte-for-byte identical to the original ROM, but our code almost certainly looks very different from the original, since it is based only on analysing the compiled code.
+
+
+## What does 100% mean?
+
+100% means that, in the first version of the game we have worked on, all of the code that can be written as compilable C code has been. It does **not** mean that the code is in any sense ready to be used easily for mods (including any ports). It also does not mean that the code is documented: there is still *a lot* of work to do on that.
+
+
+## PC port?
+
+The goal of this project is **NOT** to make any kind of port, such as to PC. Someone else might take our work and make a port (although they would be foolish to do so before the codebase is improved) but **ZeldaRET will not be involved in any porting**. A port will not happen tomorrow anyway: decomp would only be the first step.
+
+
+## What next?
+
+- Reorganisation of systems
+- Restructuring of headers
+- Documentation
+- Other versions: 17 of them!
+
+## Where can I find out more information?
+
+- You can consult the [main FAQ](faq.html).
```

Comment:
This should be modified to reflect the new structure

Reply:
This file isn't used, any more, so I'll just remove it.